dashboard/d_graphs.py

```diff
@@ -49,11 +55,116 @@ def create_scatter_graph(df: pd.DataFrame) -> alt.Chart:
         y='y:Q'
     )
 
-    final_chart = scatter_chart + zero_line
+    final_chart = zero_line + scatter_chart
```

Comment:
Nice organisational change - I assume to prevent grid lines from going over data points